src/app/model-shared/types.ts

```diff
@@ -61,12 +61,11 @@ export class PhoneColumn extends radweb.StringColumn {
     if (phone.startsWith('0')) {
       phone = getSettings(context).getInternationalPhonePrefix() + phone.substr(1);
     }
+
     if (phone.startsWith('+'))
       phone = phone.substr(1);
-    if (isDesktop())
```

Comment:
I've had this before - but I couldn't get this to work with whatsapp desktop on windows.
The advantage of using web.whatsapp - was that with whatsapp web - using this url - it opens always in the same tab - and doesn't kill the previous tabs - which was great.

Reply:
Well, I used WhatsApp Desktop for Windows, and with web.whatsapp.com - it opens the browser version which requests that I stop using the app, and with this, it allows both.
If I have the Desktop app - it uses it, if not - it allows to go to WhatsApp Web.